scripts/qa_crowbarsetup.sh

```diff
@@ -2377,6 +2377,14 @@ function custom_configuration
             if [[ $hacloud = 1 ]] ; then
                 proposal_set_value keystone default "['deployment']['keystone']['elements']['keystone-server']" "['cluster:$clusternameservices']"
             fi
+            if [[ $want_osprofiler = 1 ]] ; then
+                if iscloudver 9M10plus ; then
```

Comment:
M10 is already out and doesn't include the new osprofiler support that just merged

Reply:
@cmurphy Hm. What do I need to add here? `9GMC1plus` ?

Reply:
@bmwiedemann I guess you know the answer?

Reply:
how about `iscloudver 9M11plus`
https://github.com/SUSE-Cloud/automation/blob/master/scripts/lib/mkcloud-common.sh#L538 seems to consider any non-M* notation to be equivalent to M999

Reply:
Done. Thanks !